addon/components/nypr-account-forms/login.js

```diff
@@ -47,7 +53,29 @@ export default Component.extend({
     },
     loginWithFacebook() {
       get(this, 'session').authenticate('authenticator:torii', 'facebook-connect')
-      .catch(() => {});
+      .then(() => {
+        if (this.get('session').get('isNewSocialUser') === true) {
```

Comment:
so is this still required? it's more straightforward to just so that flash message on sign up screens, which would eliminate these conditionals.